src/app/modules/onboarding/module.nim

```diff
@@ -38,6 +44,7 @@ type
     controller: Controller
     localPairingStatus: LocalPairingStatus
     currentFlow: SecondaryFlow
+    exportedKeys: KeycardExportedKeysDto
```

Comment:
Just to keep the record of what we discussed today.

Would be very cool to spawn a routine (to be executed in a separate thread) for the `Login`, that would call all keycard commands synchronously. Then:
- we won't need to write these complex chain of async operations
- we won't need to add such temporary props like `exportedKeys`, as it would be just a local variable inside such routine.

Reply:
Not for this PR, just thoughts.